### PR TITLE
Build using AdoptOpenJDK 8 & 11

### PR DESCRIPTION
**Pull request by marcospereira**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
 scala:
-- 2.12.7
-- 2.11.12
-- 2.13.0-M3
-jdk:
-- oraclejdk8
-- openjdk11
+  - 2.12.7
+  - 2.11.12
+  - 2.13.0-M3
```
Comment (ignasi35) [inline .travis.yml]: `M3`?

File: build.sbt [hunk 2]
```diff
@@ -14,7 +14,7 @@ import scalariform.formatter.preferences._
 //---------------------------------------------------------------
 
 val scala211 = "2.11.12"
-val scala212 = "2.12.7"
+val scala212 = "2.12.8"
 val scala213 = "2.13.0-M3"
```
Comment (ignasi35) [inline build.sbt]: `M3`?

**Comment by marcospereira** [comment]
@ignasi35 play-ws still using M3:

https://github.com/playframework/play-ws/blob/master/build.sbt#L18

We can probably publish to M5 since now there are artifacts for Akka HTTP. :-)

**Comment by ignasi35** [comment]
Cool. Let's fix on a separate PR.

**Pull request merged**